vNext/AISKU/snippet/snippet.js

```diff
@@ -1,96 +1,98 @@
-﻿var sdkInstance = "appInsightsSDK";
-window[sdkInstance] = "appInsights";
-var aiName = window[sdkInstance]; // provide non default instance name through key appInsightsSDK
-var aisdk = window[aiName] || (function (aiConfig) {
-    var appInsights = {
-        config: aiConfig
-    };
-    appInsights.initialize = true; // initialize sdk on download
+﻿(function() {
+    var sdkInstance = "appInsightsSDK";
+    window[sdkInstance] = "appInsights";
```

Comment:
As per our discussions, lets see if we can have this as a `var` instead of another `window` variable.

Reply:
yeah it works fine with one window variable, and the name can be updated to user defined.